src/Dapr.Client/DaprClient.cs

```diff
@@ -125,22 +101,15 @@ public abstract class DaprClient
         /// <typeparam name="TResponse">The type of the object in the response.</typeparam>
         /// <param name="appId">The Dapr application id to invoke the method on.</param>
         /// <param name="methodName">The name of the method to invoke.</param>  
-        /// <param name="data">Data to pass to the method</param>
-        /// <param name="metadata">A key/value pair to pass to the method to invoke.</param>
+        /// <param name="data">Data to pass to the method</param>      
+        /// <param name="httpExtension">Additional fields that may be needed if the receiving app is listening on HTTP.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken" /> that can be used to cancel the operation.</param>
-        /// <returns>A <see cref="ValueTask{T}" /> that will return the value when the operation has completed.</returns>
-        /// <remarks>If the target service is listening on http, a key value pair of http.verb and the verb must be added to the metadata parameter.
-        /// For example:
-        /// 
-        ///     ...
-        ///     myMetadata.Add("http.verb", "POST");
-        ///     daprClient.InvokeMethodAsync(...., myMetadata);
-        /// </remarks>
+        /// <returns>A <see cref="ValueTask{T}" /> that will return the value when the operation has completed.</returns>  
         public abstract ValueTask<TResponse> InvokeMethodAsync<TRequest, TResponse>(
             string appId,
             string methodName,
             TRequest data,
-            Dictionary<string, string> metadata = default,
+            Dapr.Client.Http.HTTPExtension httpExtension = null,
```

Comment:
can be set to default. 
Beginning with C# 7.1, you can use the default literal to produce the default value of a type